README.md

```diff
@@ -0,0 +1,165 @@
+</p>
+
+<p align="center">🦜 MeiliSearch script to update version and migrate data 🦜</p>
+
+**MeiliSearch Version Migration Script** is a script that migrates your MeiliSearch from a version to another more recent version without losing data nor settings.
+
+**MeiliSearch** is an open-source search engine. [Discover what MeiliSearch is!](https://github.com/meilisearch/MeiliSearch)
+
+## Table of Contents <!-- omit in toc -->
+
+- [☝️ Requirements](#-Requirements)
+- [🚗 Usage](#-Usage)
+- [🎉 Features](#-features)
+- [💔 Incompatible Versions](#-version-incompatibilities)
+- [📖 Documentation](#-documentation)
+- [⚙️ Development Workflow and Contributing](#️-development-workflow-and-contributing)
+
+## ☝️ Requirements
+
+### 1. Systemctl
+
+Currently the script only works in an environment in which MeiliSearch is running as a `systemctl` service.
+
+### 2. Correct data.ms path
+
+The MeiliSearch's `data.ms` must be stored at the following address: `/var/lib/meilisearch/data.ms`.<br>
+To ensure this is the case, MeiliSearch should have started with the following flags:`--db-path /var/lib/meilisearch/data.ms`
+
+You can check the information by looking at the file located here `cat /etc/systemd/system/meilisearch.service`.<br>
+You should find a line with the specific command used.
+
+```bash
+ExecStart=/usr/bin/meilisearch --db-path /var/lib/meilisearch/data.ms --env production
+```
+
+By default all MeiliSearch instances created using one of our cloud providers are storing `data.ms` in this repository.
+
+### 3. A Running MeiliSearch instance
+
+A MeiliSearch instance should be running before launching the script. This can be checked using the following command:
+
+```bash
+systemctl status meilisearch
+```
+
+If you don't have a running MeiliSearch instance, you can create one using one of our Cloud Providers:
+
+| Cloud Provider | Project                                                                              |
+| -------------- | :----------------------------------------------------------------------------------- |
+| DigitalOcean   | [meilisearch-digitalocean](https://github.com/meilisearch/meilisearch-digitalocean/) |
+| AWS            | [meilisearch-aws](https://github.com/meilisearch/meilisearch-aws/)                   |
+| GCP            | [meilisearch-gcp](https://github.com/meilisearch/meilisearch-gcp/)                   |
+
+<br>
+
+Alternatively, by [downloading and running MeiliSearch](https://docs.meilisearch.com/learn/getting_started/installation.html#download-and-launch) on your own server and start it as a [systemctl service](https://www.freedesktop.org/software/systemd/man/systemctl.html).
+
+## 🚗 Usage
+
+⚠️ In case of failure during the execution of the script and for security purposes, we strongly recommend [creating manually your own dump](https://docs.meilisearch.com/reference/features/dumps.html#creating-a-dump).
```

Comment:
Do we add here that the dump is created at the root of the machine `/dumps/` in case users would search for it?